packages/office-ui-fabric-react/src/utilities/dateValues/DateValues.ts

```diff
@@ -1,5 +1,6 @@
 /**
  * The days of the week
+ * {@docCategory DayOfWeek}
```

Comment:
These seem like good candidates for a combined page, rather than each being their own category